notebooks/book2/10/invere_cdf_sampling.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    plt.scatter(normal_point, 0, s=POINT_SIZE, c="g", label=r"samples $\mathcal{N}(0,1)$")
</pre><p>Should it not be N(3, 1)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>